### PR TITLE
Update v1.extracts.markdown

### PR DESCRIPTION
**Pull request by TimNiedConcur**
I believe adding the capabilities section will remove any confusion on why or how to use the Extract API. 

Thank you, 
Tim Nied

### CODE REVIEW

File: src/api-reference/common/extracts/v1.extracts.markdown [hunk 1]
```diff
@@ -18,6 +18,21 @@ layout: reference
...
+
+The Extract Web Service allows for the development of a customized process that can be used to request and extract available data objects such as “Approved” Expense Reports, Invoices and Payment Requests.
+
+Through a series of Web Service calls a Application Connector can GET the Extract Definition List, use that list to perform a POST the Extract Job of their choosing. While the job runs they would periodically GET the Extract Job Status and once “Complete” the Connector can then perform a GET to get the Extract Job File data.
+
+The series of calls that your connector will make will remain the same for each implementation of the Extract Web Service. Please refer to the Executive section of this form for a process flow diagram of these steps.
+
+Resource URI
+
+https://{InstanceURL}/api/expense/extract/v1.0
```
Comment (charlieconcur) [inline src/api-reference/common/extracts/v1.extracts.markdown:32]: We should wrap these with triple ticks to get them each on a line:

```
https://{InstanceURL}/api/expense/extract/v1.0
https://{InstanceURL}/api/expense/extract/v1.0/{DefinitionID}/job
https://{InstanceURL}/api/expense/extract/v1.0/{DefinitionID}/job/status
https://{InstanceURL}/api/expense/extract/v1.0/{DefinitionID}/job/{JobID}/file
```

**Review by charlieconcur — CHANGES_REQUESTED** [review]
Address the one piece of feedback and we are good to go.

File: src/api-reference/common/extracts/v1.extracts.markdown (hunk 1 above)
Comment (charlieconcur) [reply src/api-reference/common/extracts/v1.extracts.markdown:32]: I meant to wrap all four with tick marks as a code block.

**Review by TimNiedConcur — COMMENTED** [review]
Wrapping all URL's in triple tick marks.

**Review by charlieconcur — APPROVED** [review]

**Pull request merged**